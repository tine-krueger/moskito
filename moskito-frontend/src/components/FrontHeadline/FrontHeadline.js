import styled from 'styled-components/macro'
import PropTypes from 'prop-types'

FrontHeadline.propTypes = {
    children: PropTypes.string.isRequired
}
export default function FrontHeadline({children}) {
    return (
        <HeadlineStyled>{children}</HeadlineStyled>
    )
}

const HeadlineStyled = styled.h1`
    font-family: 'Hind Madurai';
    font-size: 1.5em;
    text-align: center;
    margin-top: 1em;

`