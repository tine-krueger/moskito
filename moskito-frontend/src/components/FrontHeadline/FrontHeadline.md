```js
<FrontHeadline children={'Front Headline'}/>
```