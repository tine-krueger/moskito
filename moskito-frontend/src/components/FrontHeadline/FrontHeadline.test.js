import { render } from '@testing-library/react'
import 'jest-styled-components'
import FrontHeadline from './FrontHeadline'

const children = 'Headline'

describe('FrontHeadline', () => {

    it('renders correctly', () => {
        const { container } = render(<FrontHeadline children={children}/>)
        expect(container).toMatchSnapshot()
    })

    it('displays the given headline', () => {
        const { getByText } = render(<FrontHeadline children={children}/>)
        expect(getByText('Headline')).toBeInTheDocument()
    })

})