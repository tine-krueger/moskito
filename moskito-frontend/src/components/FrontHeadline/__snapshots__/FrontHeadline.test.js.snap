// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FrontHeadline renders correctly 1`] = `
.c0 {
  font-family: 'Hind Madurai';
  font-size: 1.5em;
  text-align: center;
  margin-top: 1em;
}

<div>
  <h1
    class="c0"
  >
    Headline
  </h1>
</div>
`;
